### PR TITLE
Update and rename SuperHeavyBoostersHistoricalNexus.netkan to Nexus.n…

### PR DESCRIPTION
**Pull request by TiktaalikDreaming**
Possible Nexus CKAN updatey fix.  Or what I see as matching the NetKAN file to the specs suggested in the wiki notes.

### CODE REVIEW

File: NetKAN/Nexus.netkan [hunk 1]
```diff
@@ -3,11 +3,16 @@
     "spec_version": 1,
     "x_via": "Automated KerbalStuff CKAN submission",
     "license": "MIT",
-    "identifier": "SuperHeavyBoostersHistoricalNexus",
+    "x_netkan_license_ok": true
```
Comment (techman83) [inline NetKAN/Nexus.netkan]: Missing a comma here.

**Comment by TiktaalikDreaming** [comment]
comma added.  :-/

File: NetKAN/Nexus.netkan [hunk 2]
```diff
@@ -3,11 +3,15 @@
...
     "license": "MIT",
-    "identifier": "SuperHeavyBoostersHistoricalNexus",
+    "identifier": "Nexus",
     "install": [
         {
-            "file": "Nexus",
+            "file": "GameData/Nexus",
             "install_to": "GameData"
         }
     ]
```
Comment (techman83) [inline NetKAN/Nexus.netkan]: Sorry I missed this one too.

**Comment by plague006** [comment]
`identifier` does not change unless under extreme circumstances. It's a non user-visible ID for mods themselves that CKAN requires to remain static. If you want to change what name users see for the mod, you can hardcode the `name` field or change the mod's name via the KerbalStuff page.

I'm merging #2276 because it's not touching the `identifier` field and I'm creating a separate PR for the dependency you wanted to add.

Closing this with #2276.

**Pull request closed without merging**